scalyr_agent/configuration.py

```diff
@@ -1115,16 +1115,15 @@ def __verify_main_config_and_apply_defaults(self, config, file_path, apply_defau
             config, 'k8s_log_api_responses_to_disk', False, description, apply_defaults, env_aware=True
         )
 
-        # TODO-163 : make controlled cache warmer settings more aggressive
-        # Optional (defaults to 5). The maximum number of temporary errors that may occur when warming a pod\'s entry,
-        # before the warmer blacklists it
+        # Optional (defaults to 3). The number of times the warmer will retry a query to warm a pod before giving up and
+        # classifying it as a Temporary Error
         self.__verify_or_set_optional_int(
-            config, 'k8s_controlled_warmer_max_attempts', 5, description, apply_defaults, env_aware=True
+            config, 'k8s_controlled_warmer_max_query_retries', 3, description, apply_defaults, env_aware=True
         )
-        # Optional (defaults to 3). The number of times the warmer will retry a query to warm a pod if it fails due to
-        # a temporary error.
+        # Optional (defaults to 5). The maximum number of Temporary Errors that may occur when warming a pod's entry,
+        # before the warmer blacklists it.
         self.__verify_or_set_optional_int(
-            config, 'k8s_controlled_warmer_max_query_retries', 3, description, apply_defaults, env_aware=True
+            config, 'k8s_controlled_warmer_max_attempts', 5, description, apply_defaults, env_aware=True
```

Comment:
Reordered these 2 config params and improved description to make them easier to understand.